data/mods/BombasticPerks/perks.json

```diff
@@ -290,6 +290,15 @@
     "category": [ "perk" ],
     "enchantments": [ { "condition": "ALWAYS", "values": [ { "value": "EVASION", "add": 0.05 } ] } ]
   },
+  {
+    "type": "mutation",
+    "id": "perk_long_shot",
+    "name": { "str": "Long shot" },
```

Comment:
```suggestion
    "name": { "str": "Long Shot" },
```

Reply:
A bunch aren't fully capitalized, I'll do a standardizing in another PR.